htons_.c

```diff
@@ -0,0 +1,47 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ *
+ * Authors: Marius-Cristian Baciu <marius.baciu@stud.acs.upb.ro>
+ *
+ *
+ * Copyright (c) 2020, NEC Europe Ltd., NEC Corporation. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2020, University Politehnica of Bucharest. All rights reserved.
```

Same reason as in the `gluecode.c` file.

Reply:
Done